locales/ki.json

```diff
@@ -0,0 +1,35 @@
+{
+	"Welcome to barefoot nomad": "Murakaza neza kuri barefoot nomad",
+	"Email already registered": "Imeri yamaze kwiyandikisha",
+	"User registered": "Umukoresha yiyandikishije",
+	"Authentication failed. User not found.": "Kwemeza byarananiranye. Umukoresha ntabwo yabonetse.",
+	"Authentication failed. Wrong password.": "Kwemeza byarananiranye. Ijambobanga ritari ryo.",
+	"Error": "Ikosa",
+	"No User with the specified": "Nta mukoresha ufite ibyasobanuwe",
+	"User update failed, a user with the specified email exist": "Kuvugurura kwabakoresha byarananiranye, umukoresha ufite imeri yihariye irahari",
+	"User updated successfully": "Umukoresha agevuguruwe neza",
```

Comment:
you can say _umukoresha yavuguruwe neza_

Reply:
No, because the action was happening at that moment, it's obvious of being in present. It should be that it the message was ` User was updated successfully`